django/db/models/query.py

```diff
@@ -322,6 +322,7 @@ def aggregate(self, *args, **kwargs):
         """
         if self.query.distinct_fields:
             raise NotImplementedError("aggregate() + distinct(fields) not implemented.")
+        self._validate_values_are_expressions(args + tuple(kwargs.values()), 'aggregate')
```

Comment:
It might help readability to pass `method_name='aggregate'` -- otherwise the meaning of that string isn't so clear without looking up the method's signature.